nrpy/infrastructures/BHaH/cmdline_input_and_parfiles.py

```diff
@@ -287,7 +287,7 @@ def register_CFunction_cmdline_input_and_parfile_parser(
 """
     if found_REAL:
         prefunc += r"""
-static void read_double(const char *value, double *result, const char *param_name) {
+static void read_double(const char *value, REAL *result, const char *param_name) {
```

Comment:
Function should be renamed read_REAL() or something like that...

Reply:
Resolved in 902595f4e8521eeb2810c499ff088c083d5c0fc0